pkg/apis/core/validation/seed_test.go

```diff
@@ -871,8 +871,9 @@ var _ = Describe("Seed Validation Tests", func() {
 
 				Expect(errorList).To(ConsistOf(
 					PointTo(MatchFields(IgnoreExtras, Fields{
-						"Type":  Equal(field.ErrorTypeForbidden),
-						"Field": Equal("spec.settings.topologyAwareRouting.enabled"),
+						"Type":   Equal(field.ErrorTypeForbidden),
+						"Field":  Equal("spec.settings.topologyAwareRouting.enabled"),
+						"Detail": Equal("topology-aware routing can only be enabled on multi-zone Seed clusters (with at least two zones in spec.provider.zones)"),
```

Comment:
detail doesn't match error message below

Reply:
Can you elaborate? I guess you were confused as this is a small addition to the existing test for the Seed validation. It is not operator related test, hence I would say it is as expected.